src/app/loadout-builder/process/useProcess.ts

```diff
@@ -222,20 +228,27 @@ const groupComparator = chainComparator(
 );
 
 /**
- * This groups items for process depending on whether any general, other or raid mods are locked as follows
- * - If there are general, other or raid mods locked it groups items by (stats, masterworked, modSlot, energyType).
- * - If there are only general mods locked it groupes items by (stats, masterwork, energyType)
- * - If no general, other or raid mods are locked it groups by (stats, masterworked).
+ * To reduce the number of items sent to the web worker we group items by a number of varying
+ * parameters, depending on what mods and armour upgrades are selected.
+ *
+ * After items have been grouped we only send a single item (the first one) as a representitive of
```

Comment:
```suggestion
 * After items have been grouped we only send a single item (the first one) as a representative of
```

Reply:
Hmmm why isn't that spell checker picking me up on this.